hikari/internal/ux.py

```diff
@@ -95,6 +95,11 @@ def init_logging(
         If `builtins.True`, return `builtins.True` always, otherwise only
         return `builtins.True` if the device supports colour output and the
         `allow_color` flag is not `builtins.False`.
+
+    Raises
+    ------
+    builtins.ValueError
+        Default $-substitution was overridden.
```

Comment:
Think this is meant to go bellow instead of here :)

Reply:
Can you clarify what I should do?

Reply:
oh...

Reply:
it has been fixed :pensive: